badges/badges/settings.py

```diff
@@ -205,4 +205,12 @@ def skip_suspicious_operations(record):
 ##################################################################
 # Testing mode
 #################################################################
-TEST = 'test' in sys.argv
\ No newline at end of file
+TEST = 'test' in sys.argv
```

Comment:
I don't have a mayor pain with this, but it's consider bad practice to test for testing in config (or anywhere in code). Let me know if you want to talk and figure out how to avoid it.